pkg/codegen/nodejs/gen.go

```diff
@@ -107,6 +107,13 @@ func (mod *modContext) tokenToType(tok string, input bool) string {
 	components := strings.Split(tok, ":")
 	contract.Assertf(len(components) == 3, "malformed token %v", tok)
 
+	if mod.tool == "crd2pulumi" && components[0] == "crds" {
```

Comment:
Yeah, I do wonder if we should add some type of flag to opt-in to this behavior, rather than looking at the tool name, but, I'm ok with this for now.